llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -642,6 +642,7 @@ define void @foo() {
   // Check classof() and creation.
   auto *Poison = cast<sandboxir::PoisonValue>(I0->getOperand(0));
   EXPECT_EQ(Poison->getType(), Int32Ty);
+  EXPECT_TRUE(isa<sandboxir::UndefValue>(Poison)); // Poison is Undef
```

Comment:
same problem?

Reply:
Hmm what do yo mean? This just checks that a Poison is also an Undef.

Reply:
This is correct.

Reply:
LGTM, Poison value is an undef.